CLONE_RESTORE.md

```diff
@@ -0,0 +1,200 @@
+<h1>Creating and Restoring in-cluster Postgres Database Backups in Harboor</h1>
+
+<h2>Why do we want to  create backups</h2>
+- To be able to recover harbor indexed data in case of disaster so we can still have the list of cached artifacts stored in harbor
+
+There are currently two ways to configure a database in harbor
+1. Standard database configurations can be used to set the existing pre-deployed or cloud database services as the dependent database of the deploying Harbor.
+2. The in-cluster database configuration can be configured to let the <b>Harbor operator automatically deploy an in-cluster PostgreSQL database service using the zalando postgres operator</b> with HA supported as the dependent database of the deploying Harbor. 
+
+The second method is the preferred method because removes the need to manage a separate postgres instance.
+
+<h2> Creating and storing backups in S3 using WALG </h2> 
+In other to create backups stored in s3 the following fields must be set in the `pod-config` configmap the postgres operator will read environment variables from.
+
+```
+USE_WALG_BACKUP: "true"
+USE_WALG_RESTORE: "true"
+BACKUP_NUM_TO_RETAIN: "14" # For 2 backups per day, keep 7 days of base backups
+BACKUP_SCHEDULE: "*/10 * * * *"
+AWS_ACCESS_KEY_ID: < * >
+AWS_SECRET_ACCESS_KEY: < * >
+AWS_REGION: < * >
+WALG_DISABLE_S3_SSE: "true"
+WALE_DISABLE_S3_SSE: "true"
+WAL_S3_BUCKET: < * > # Required, the bucket the backups will be created in
```

Comment:
Have the following `harbor-operator` logical_backup properties been tested as alternative method for configuring backups to s3?

Reply:
Yeah, we looked at it but it doesn't meet our [use-case](https://opensource.zalando.com/postgres-operator/docs/administrator.html#logical-backups). Also, it requires the `enableLogicalBackup` parameter to be set to true which isn't currently in Harbor operator